internal/api/v1/application/deploy.go

```diff
@@ -299,8 +313,8 @@ func replaceInternalRegistry(ctx context.Context, cluster *kubernetes.Cluster, i
 	if err != nil {
 		return imageURL, err
 	}
-	// If there is a local registry and kube is not supposed to access it through Ingress
-	if localURL != "" && !viper.GetBool("force-kube-internal-registry-tls") {
+
+	if localURL != "" {
```

Comment:
This is everything the epinio server has to know. The `force-kube-internal-registry-tls` is now only an instruction to the installer. The server doesn't care about that flag.